RecoLocalCalo/EcalRecProducers/plugins/alpaka/EcalUncalibRecHitProducerPortable.cc

```diff
@@ -73,18 +83,66 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
     desc.add<double>("outOfTimeThresholdGain61mEE", 1000);
     desc.add<double>("amplitudeThresholdEB", 10);
     desc.add<double>("amplitudeThresholdEE", 10);
+
+    desc.add<std::vector<double>>("EBtimeFitParameters",
+                                  {-2.015452e+00,
+                                   3.130702e+00,
+                                   -1.234730e+01,
+                                   4.188921e+01,
+                                   -8.283944e+01,
+                                   9.101147e+01,
+                                   -5.035761e+01,
+                                   1.105621e+01});
+    desc.add<std::vector<double>>("EEtimeFitParameters",
+                                  {-2.390548e+00,
+                                   3.553628e+00,
+                                   -1.762341e+01,
+                                   6.767538e+01,
+                                   -1.332130e+02,
+                                   1.407432e+02,
+                                   -7.541106e+01,
+                                   1.620277e+01});
+    desc.add<std::vector<double>>("EBamplitudeFitParameters", {1.138, 1.652});
+    desc.add<std::vector<double>>("EEamplitudeFitParameters", {1.890, 1.400});
+
     desc.addUntracked<std::vector<uint32_t>>("kernelMinimizeThreads", {32, 1, 1});
     desc.add<bool>("shouldRunTimingComputation", true);
+
     confDesc.addWithDefaultLabel(desc);
   }
 
-  EcalUncalibRecHitProducerPortable::EcalUncalibRecHitProducerPortable(const edm::ParameterSet& ps)
+  std::unique_ptr<EcalMultifitParametersCache> EcalUncalibRecHitProducerPortable::initializeGlobalCache(
+      edm::ParameterSet const& ps) {
+    PortableHostObject<EcalMultifitParameters> params(cms::alpakatools::host());
+
+    auto const ebTimeFitParamsFromPSet = ps.getParameter<std::vector<double>>("EBtimeFitParameters");
```

Comment:
```suggestion
    auto const& ebTimeFitParamsFromPSet = ps.getParameter<std::vector<double>>("EBtimeFitParameters");
```
?

etc.

Reply:
(though I see there were like that in the original code as well)

Reply:
`ParameterSet::getParameter()` returns by value, so capturing the returned object by `const` reference instead of by value won't save any resources. (but I _can_ change these if preferred)